src/components/SocialLinks.vue

```diff
@@ -4,7 +4,7 @@
       <a href="https://github.com/ekinkaradag" target="_blank">
         <img
           class="icon"
-          :src="getIconUrl('github-logo')"
+          :src="getIconUrl('linkedin-logo')"
```

Comment:
probably a mistake. Revert

```suggestion
          :src="getIconUrl('github-logo')"
```